gensim/models/ldamodel.py

```diff
@@ -27,6 +27,50 @@
 
 .. [1] http://www.cs.princeton.edu/~mdhoffma
 
+
+Examples
+--------
+
+#. Train an LDA model using a gensim corpus:
+
+>>> from gensim.test.utils import common_corpus
+>>> from gensim.corpora.dictionary import Dictionary
+>>>
+>>> lda = LdaModel(common_corpus, num_topics=10)  # train model
+
+#. Save a model to disk, or reload a pretrained model.
+
+>>> from gensim.test.utils import datapath
+>>>
+>>> # Save model to disk.
+>>> temp_file = datapath("model")
+>>> lda.save(temp_file)
+>>>
+>>> # Load a potentially pretrained model from disk.
+>>> lda = LdaModel.load(temp_file)
+
+#. Query, the model using new, unseen documents:
+
+>>> other_texts = [
+...     ['computer', 'time', 'graph'],
+...     ['survey', 'response', 'eps'],
+...     ['human', 'system', 'computer']
+... ]
+>>> other_dictionary = Dictionary(other_texts)
```

Comment:
This is a **serious** mistake, you should use only one dictionary for vectorization: here you should re-use dictionary that was used to create `common_corpus` or (better) create it yourself from start, convert `common_texts`, etc.

Same mistake in other files.